modules/client-handler/src/main/java/org/apache/ignite/client/handler/requests/table/ClientTupleGetAndDeleteRequest.java

```diff
@@ -46,6 +47,6 @@
         var table = readTable(in, tables);
         var tuple = readTuple(in, table, true);
 
-        return table.getAndDeleteAsync(tuple).thenAccept(resTuple -> writeTuple(out, resTuple));
+        return table.getAndDeleteAsync(tuple).thenAccept(resTuple -> writeTuple(out, resTuple, TuplePart.VAL));
```

Comment:
Do you plan to reuse this (and similar) methods in Table binary view?
I think user expects TableView.get(t) will return full row (key+value parts), but KvView will return "value" part only, thought.
Shouldn't TuplePart be passed from outside?

Reply:
> Do you plan to reuse this (and similar) methods in Table binary view?

This is a server-side handler. `Table` and `KvView` APIs only differ in the way they return data to the user, but the data is the same. There is nothing specific to `Table` or `KvView` on the protocol level - all operations are shared.

For `Get*` operations we only return the value part from the server, because the client already knows the key. For the `Table` view, we combine returned value columns with known key columns to create the full row and return it to the user.

Reply:
I've updated the PR description to clarify this point.